extension-localization/src/main/java/com/mapbox/maps/extension/localization/Localization.kt

```diff
@@ -25,6 +33,21 @@ internal fun setMapLanguage(locale: Locale, style: StyleInterface, layerIds: Lis
     return
   }
 
+  /**
+   * Check if the server side localization is set and Log the error if present.
+   * `localizeLabel` method (ie Style localization) will not work with server side localization
+   */
+  checkServerSideLocalizationSet { isLocalizationSet ->
+    if (isLocalizationSet) {
```

Comment:
If it doesn't work - should we execute the logic below? Shouldn't it be skipped instead?